Cargo.toml

```diff
@@ -19,7 +19,8 @@ num = "0.4.0"
 rand = "0.8.1"
 serde = { version = "1.0.118", features = ["derive"] }
 sha2 = "0.9.2"
-zbus = "1.7.0"
-zbus_macros = "1.7.0"
-zvariant = "2.4.0"
-zvariant_derive = "2.4.0"
+zbus = "2"
+zbus_names = "2"
+zbus_macros = "2"
+zvariant = "3"
+zvariant_derive = "3"
```

Comment:
you can use those directly from zbus iirc

Reply:
AFAICS that isn't compatible with `zvariant_derive` (which isn't exported from `zbus`):

```
error[E0433]: failed to resolve: could not find `zvariant` in the list of imported crates
  --> src/proxy/collection.rs:58:41
   |
58 | #[derive(Debug, Serialize, Deserialize, Type)]
   |                                         ^^^^ could not find `zvariant` in the list of imported crates
   |
   = note: this error originates in the derive macro `Type` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Which happens even when I do `use zbus::zvariant`. I can remove `zbus_maros` though, thanks for the heads up.

Reply:
Right I noticed this issue yesterday as well. IIRC, it works if you:

```rust
use zbus::zvariant;
use zvariant::Type;
```

Will see how we can solved this.

Reply:
Actually NM my last comment, that doesn't work either. :( However, for the workaround I'd suggest adding dep only only `zvariant` and then using macros from there directly:

```rust
use zvariant::Type;
```

macro crate only exists cause Rust wants macros in a separate crate.

Reply:
Done, thanks

Reply:
Fix is [coming soon](https://gitlab.freedesktop.org/dbus/zbus/-/merge_requests/456/diffs?commit_id=5c1d4c9807ae56db236119a0bd7cfc2f91edf3bd) in the next zvariant release.